npm/binary.js

```diff
@@ -20,17 +20,22 @@ const getPlatform = () => {
 };
 
 const getBinaryURL = (version, platform) => {
-  const site = process.env.WRANGLER_BINARY_HOST ||
-      process.env.npm_config_wrangler_binary_host ||
-      'https://workers.cloudflare.com/get-npm-wrangler-binary';
+  const site =
+    process.env.WRANGLER_BINARY_HOST ||
+    process.env.npm_config_wrangler_binary_host ||
+    "https://workers.cloudflare.com/get-npm-wrangler-binary";
```

Comment:
is there an issue for adding a custom host?

Reply:
#1286

Reply:
but that issue is about wrangler's installation path not changing the endpoint to get the executable